youtube_dl/extractor/prosiebensat1.py

```diff
@@ -375,7 +375,9 @@ def _extract_clip(self, url, webpage):
         title = self._html_search_regex(self._TITLE_REGEXES, webpage, 'title')
         info = self._extract_video_info(url, clip_id)
         description = self._html_search_regex(
-            self._DESCRIPTION_REGEXES, webpage, 'description', fatal=False)
+            self._DESCRIPTION_REGEXES, webpage, 'description', default=None)
+        if description is None: 
+            description = self._og_search_description(webpage, default=None)
```

Comment:
Better to use fatal=False instead of default=None here.

For optional fields, if all extraction approaches failed, youtube-dl should issue a warning.